### ISSUE TITLE
Can't save edits after using weather widget

### ISSUE THREAD

**Issue by tnttruong** [joelacus/Cobalt#17]
Whenever the weather widget is placed, any further edits can no longer be saved to that specific workspace even after browser refresh. You can only save edits to a new workspace until the weather widget is used again. 

Have not tested with other widgets so it might not just be a weather widget issue

**Comment by joelacus** [comment]
Oh, that's odd, sorry about that. Maybe the saving function is unable to find something on the page it's expecting to while it's trying to create the save, and thus failing.

Does it save ok if you never add the weather widget, or remove it?

If you open the dev console (F12), and go the console tab, it might show an error.

Also, which browsers and browser versions are you guys using?

I'm abroad at the moment and won't be back home until later this week, but when I am, I'll try and figure out what's wrong and fix the issue.

**Comment by SteeboGreebo** [comment]
I don't have the weather widget, only some shortcuts and notes. I am on Chrome and v0.10.2.
When simply opening the setting tab and pressing save, the following error is thrown:

```
index-e46d82b3caa3443b09d0.js:2 Uncaught TypeError: Cannot read properties of undefined (reading 'replaceAll')
    at index-e46d82b3caa3443b09d0.js:2:127381
    at index-e46d82b3caa3443b09d0.js:2:128033
    at NodeList.forEach (<anonymous>)
    at m (index-e46d82b3caa3443b09d0.js:2:123703)
    at d (index-e46d82b3caa3443b09d0.js:2:123430)
    at index-e46d82b3caa3443b09d0.js:2:498202
    at HTMLDivElement.<anonymous> (index-e46d82b3caa3443b09d0.js:2:503081)
    at HTMLDocument.dispatch (vendor-8580b2fec7e178a27892.js:2:278419)
    at m.handle (vendor-8580b2fec7e178a27892.js:2:276410)
```

**Comment by joelacus** [comment]
Ah right, I think I know what is wrong actually. If you can hold on for a few more days, I'll publish an update as soon as I can. Sorry again for the issue

**Comment by joelacus** [comment]
Hi. I've just published version 0.10.3 with a fix for this. Let me know if you still have any issues, thanks.

**Comment by Khosro1986** [comment]
My cobalt version is still Version 0.10.2
Since you have said, I have just published, it means version 0.10.3 is now available or should we wait for it?

**Comment by joelacus** [comment]
I've uploaded the update, but it still needs to be approved by Google and Microsoft. It may take a few days, but you'll get the update automatically when they're done. Mozilla has already approved it for Firefox though and is available now.
